apps/main/src/llamalend/features/market-list/cells/RateCell/RewardsTooltipItems.tsx

```diff
@@ -1,59 +0,0 @@
-import lodash from 'lodash'
```

Comment:
Darnit I refactored this in #1446, could've just deleted it 😭

Reply:
:laughing: too bad